index.js

```diff
@@ -0,0 +1,21 @@
+export async function theme({Quasar, brand}) {
+  const variables = {};
+  Object.keys(brand).forEach(colorName => {
+    Quasar.utils.colors.setBrand(colorName, brand[colorName]);
+    variables[`q-color-${colorName}`] = brand[colorName];
```

Comment:
Color tags like "primary" and "secondary" will work, but the quasar setBrand has special support for some others.  It does make this code complicated but they should either be supported here in the same way or at least throw a warning about the issue.
https://github.com/quasarframework/quasar/blob/dev/src/utils/colors.js#L193-L213